api/routes.py

```diff
@@ -69,40 +69,6 @@ async def read_price_data_zone(price_data_zone: str, db: Session = Depends(get_d
         "time_end": price_data.time_end
     }
 
-# GET endpoint: Function to get price levels by zone
-@router.get("/price-levels/{zone}")
-async def get_price_levels(zone: str, db: Session = Depends(get_db)):
-    print(f"Fetching latest price entry for zone: {zone}")
-
-# GET endpoint: Fetch data by Zone And time interval (Seconds since epoch)
-@router.get("/price-data")
```

Comment:
This endpoint is being used by the client, the client asks for the date range so that it can get all data points for the graph